### DISCUSSION TITLE
The provided connection string is invalid or server is unreachable

### DISCUSSION THREAD

**Discussion by turbobuggames** [OrchardCMS/OrchardCore#13941] [General]
I m trying to set up OrchardCore for the first time. When I run it from Visual Studio, I get to the setup page in my browser.
I enter in the details and I get the error "The provided connection string is invalid or server is unreachable".

I have tried so many different connection strings to try to get it to work (windows, sql logins, different variations of localhost etc). I know it is likely something I am doing wrong in SQL Server or in the connection string.

DB & OrchardCore App is saved on my D Drive. SQL Server instance connected in SSMS.
I have my windows user as the owner of DB.
OrchardCore user has permissions in DB also.

Here are a few examples of what I have tried (there have been many other combinations):
``
Server=localhost\MSSQLSERVER02;Database=OrchardTest;Trusted_Connection=True;
``
``Server=localhost;Database=OrchardTest;Trusted_Connection=True;MultipleActiveResultSets=true``
``Server=DESKTOP-3FOI0TC\MSSQLSERVER02;Database=OrchardTest;Integrated Security=true;``
``Server=DESKTOP-3FOI0TC;Database=OrchardTest;User Id=OrchardCore;Password=yyyuihs6gk$ysihsi;``

I'd really like to try to get it working with SQL Server and not SQLite.

Does anyone have any advice on what I might be doing wrong? 

Cheers

**Comment by jtkech** [comment]
See https://docs.orchardcore.net/en/latest/docs/releases/1.5.0/#yessql-breaking-changes
You now need to add Encrypt=false or TrustServerCertificate=True

**Reply by turbobuggames** [reply]
Thank you jtkech! That worked!

**Reply by hishamco** [reply]
This is annoying, it should be fixed in #13792
